xml/System/Guid.xml

```diff
@@ -1081,7 +1096,7 @@ After trimming, the length of the read-only character span is 0.
         </exception>
       </Docs>
     </Member>
-    <Member MemberName="Parse">
+    <MemberGroup MemberName="Parse">
```

Comment:
```suggestion
    <Member MemberName="Parse">
```

Reply:
You need to create a new entry for the MemberGroup like here:
https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Guid.xml#L698